wuvt/trackman/mail.py

```diff
@@ -56,3 +57,29 @@ def send_playlist(djset, tracks):
         current_app.logger.warning(
             "Trackman: Failed to send email for DJ set {0}: {1}".format(
                 djset.id, exc))
+
+def send_chart(chart):
+    print(chart)
+
+    msg = MIMEMultipart('alternative')
+    msg['Date'] = email.utils.formatdate()
+    msg['From'] = current_app.config['MAIL_FROM']
+    msg['To'] = current_app.config['CHART_DEST']
+    msg['Message-Id'] = email.utils.make_msgid()
+    msg['X-Mailer'] = "Trackman"
+    timestamp = datetime.now()
+    msg['Subject'] = u"[New Music Chart] {0}".format(
```

Comment:
We need to include `TRACKMAN_NAME` so we know which instance this came from. I would prefer `[{TRACKMAN_NAME}] New Music Chart: {timestamp}` instead.